src/richie-front/js/components/SearchFilterGroup/SearchFilterGroup.tsx

```diff
@@ -14,7 +14,6 @@ export interface SearchFilterGroupProps {
 
 export const SearchFilterGroup = (props: SearchFilterGroupProps) => {
   const { humanName, values } = props.filter;
-
```

Comment:
no idea why there is this modification. Probably the linter who did that.

Reply:
Then there's probably no point in keeping it in the commit :)